.github/workflows/rust.yml

```diff
@@ -21,128 +57,88 @@ concurrency:
 
 jobs:
   check:
-    runs-on: ubuntu-latest
     name: check
+    runs-on:
+      group: 8-cores_32GB_Ubuntu Group
     timeout-minutes: 60
-    env:
-      RUSTFLAGS: -D warnings
```

Comment:
Why we've removed `RUSTFLAGS: -D warnings` ? 

It will allow warnings to sneak in

Reply:
I've enabled it globally instead, under `env:`

Reply:
Oh, I see! 

I think that's good. Maybe a little strict, since bad formatting will fail tests, even if they are correct, giving a less information. But on the other hand having it global prevents forgetting it if new job added.

Reply:
Oh, you're right, I didn't even realize. I think it doesn't matter much, but we can remove it from tests if the next CI person has a preference!